app/src/main/res/layout/activity_main.xml

```diff
@@ -5,15 +5,190 @@
+        app:layout_constraintStart_toStartOf="@id/guideline_start"
+        app:layout_constraintTop_toBottomOf="@id/top_view"
+        app:layout_constraintVertical_bias="0.95"
+        app:strokeColor="@color/black"
+        app:strokeWidth="2dp" />
+
+    <androidx.constraintlayout.helper.widget.Flow
+        android:id="@+id/links"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        app:constraint_referenced_ids="imageButtonFacebook,imageButtonLinkedIn,imageButtonInstagram"
+        app:flow_firstHorizontalBias="0.5"
+        app:flow_firstHorizontalStyle="spread_inside"
+        app:flow_horizontalGap="16dp"
+        app:layout_constraintBottom_toTopOf="@id/edit_profile"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+        app:layout_constraintTop_toBottomOf="@id/top_view" />
+
+    <androidx.appcompat.widget.AppCompatImageButton
+        android:id="@+id/imageButtonFacebook"
+        style="@style/ImageButtonStyle"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:contentDescription="@string/facebook_description"
+        app:layout_constraintBottom_toBottomOf="@id/links"
+        app:layout_constraintTop_toTopOf="@id/links"
+        app:srcCompat="@drawable/facebook" />
+
+    <androidx.appcompat.widget.AppCompatImageButton
+        android:id="@+id/imageButtonLinkedIn"
+        style="@style/ImageButtonStyle"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:contentDescription="@string/linkedin_description"
+        app:layout_constraintBottom_toBottomOf="@id/links"
+        app:layout_constraintTop_toTopOf="@id/links"
+        app:srcCompat="@drawable/linkedin" />
+
+    <androidx.appcompat.widget.AppCompatImageButton
+        android:id="@+id/imageButtonInstagram"
+        style="@style/ImageButtonStyle"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:contentDescription="@string/instagram_description"
+        app:layout_constraintBottom_toBottomOf="@id/links"
+        app:layout_constraintTop_toTopOf="@id/links"
+        app:srcCompat="@drawable/instagram" />
+
+
+    <de.hdodenhof.circleimageview.CircleImageView
+        android:id="@+id/icon_profile"
+        android:layout_width="113dp"
+        android:layout_height="0dp"
+        android:src="@drawable/icon1"
+        app:layout_constraintBottom_toBottomOf="@id/top_view"
+        app:layout_constraintDimensionRatio="1:1"
+        app:layout_constraintEnd_toEndOf="@id/guideline_end"
+        app:layout_constraintStart_toStartOf="@id/guideline_start"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.35" />
+
+    <androidx.appcompat.widget.AppCompatTextView
```

Comment:
The name and the profession are the abstract group. They have a vertical margin of 8dp between them.

Reply:
done